ivy_tests/test_ivy/test_frontends/test_paddle/test_math.py

```diff
@@ -473,6 +473,35 @@ def test_paddle_atanh(
     )
 
 
+# broadcast_shape
+@handle_frontend_test(
+    fn_tree="paddle.broadcast_shape",
+    dtype_and_x=helpers.dtype_and_values(
+        available_dtypes=helpers.get_dtypes("valid"),
```

Comment:
please add a `num_arrays=2` configuration here for generating 2 arrays else the tests are raising an index out of range error when accessing `y=x[1]` thanks!